dnf/cli/commands/__init__.py

```diff
@@ -274,12 +274,7 @@ def configure(self):
         _checkEnabledRepo(self.base)
 
     def run(self):
-        query = self.base.sack.query().filterm(upgrades_by_priority=True)
-        if self.base.conf.obsoletes:
```

Comment:
Doesn't removal of this condition change the behavior in terms of sensitivity to conf.obsoletes option?

Reply:
I believe that the same mechanism is perform at base.check_updates(). Therefore with new workflow it is required only once.